torchaudio/transforms.py

```diff
@@ -643,6 +643,11 @@ class Resample(torch.nn.Module):
             cached as ``torch.float64``. If you use resample with lower precision, then instead of providing this
             providing this argument, please use ``Resample.to(dtype)``, so that the kernel generation is still
             carried out on ``torch.float64``.
+
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalization=True)
+        >>> resampler = transforms.Resample(sample_rate, sample_rate/10)
```

Comment:
Rename ``resampler`` as ``transform`` so that we have all examples consistent.

Reply:
Sure thing!